app/package.json

```diff
@@ -2,15 +2,16 @@
   "name": "headlamp",
   "version": "0.2.0",
   "description": "",
-  "main": "electron/main.js",
+  "main": "electron/src/main.js",
```

Comment:
We don't have a `main` defined in the frontend. Not sure if we need it here either (it's fine if we do though).
Please notice that below you still have places showing `build/electron/main.js`.

Reply:
Perhaps just change it to electron/src/main.ts ? (note the ts). It's kind of a nice documentation for the entry point.

https://docs.npmjs.com/cli/v6/configuring-npm/package-json#main

Reply:
main entry tells the electron cli where to look for the entry file for electron codebase and that's why i changed it to ```electron/src/main.js``` because this is where we have the compiled js code from typescript.
about ```build/electron/main.js``` that is used by electron-builder and its a place where we copy the electron codebase while the build step so it doesn't require the change.